### PR TITLE
Fix/faq search

### PR DESCRIPTION
**Pull request by lollomag**

### CODE REVIEW

**Review by emnlpstr — APPROVED** [review]

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ questions:
     sectionId: investimenti
     accordions:
       - title: Quali sono gli investimenti di Italia digitale 2026?
-        content: Next Generation EU (NGEU) è un piano da €750 miliardi di Euro che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
+        content: testo due
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:40]: ??

File: contents/faq-page/faq.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ questions:
     sectionId: supporto
     accordions:
       - title: È possibile richiedere supporto?  
-        content: Next Generation EU (NGEU) è un piano da €750 miliardi di Euro che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
+        content: Proviamo il filtro è un piano da €750 miliardi di Euro che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
       - title: Come posso ricevere aggiornamenti? 
         content: Next Generation EU (NGEU) è un piano da €750 miliardi di Euro che ha l'obiettivo di rilanciare l'economia europea dopo la pandemia di COVID-19 e di renderla più verde e digitale.
 noResults: Nessun risultato trovato
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:67]: same as above, why this?

File: src/layouts/FaqPage.jsx [hunk 3]
```diff
@@ -33,92 +35,74 @@ export const FaqPage = () => {
...
-        faq.questions.forEach((question) => {
-          if (getAccordionsFiltered(question).length) {
-            newQuest.push(question);
-          }
-        });
-        setQuestions(newQuest);
-      } else {
-        setQuestions(faq.questions);
-      }
+      console.log(inputValue);
```
Comment (sebbalex) [inline src/layouts/FaqPage.jsx:81]:
```suggestion

```

File: src/layouts/FaqPage.jsx [hunk 4]
```diff
@@ -33,92 +35,74 @@ export const FaqPage = () => {
...
+      setInputValue('');
+      setQuestions(faq.questions);
+      console.log('dopo set', inputValue);
+      
+      // if (inputValue && inputValue.length >= 3) {
+
+      //   setQuestions(getNewQuestions(inputValue));
+      // } else {
+      //   setQuestions(faq.questions);
+      // }
```
Comment (sebbalex) [inline src/layouts/FaqPage.jsx:91]:
```suggestion

```

**Review by sebbalex — CHANGES_REQUESTED** [review]

**Pull request merged**